src/record.h

```diff
@@ -21,6 +21,7 @@ class Record
     FLAG_LIGHT           = 0x00000200,  // SSE & FO4 light flag (FE/FF memory space)
     FLAG_OVERLAY = 0x00000200,  // SF overlay flag (does not claim new memory space,

```

Comment:
Maybe update the comment to explain that this is not used anymore?

Reply:
I believe the flag technically still exists it just has no real use at the moment. Nothing supports it and it doesn't function as initially expected. (Unless they fixed it in this update but I haven't heard about it if so.)